src/kepler_model/cmd/main.py

```diff
@@ -406,6 +406,7 @@ def train_from_data(args):
 - --energy-source : specify target energy sources (use comma(,) as delimiter) 
 - --thirdparty-metrics : specify list of third party metric to export (required only for ThirdParty feature group)
 - --id : specify machine ID 
+- --vm-train: specify whether to use vm feature and energy metrics for training - true: use vm feature metrics. Default is false
```

Comment:
As indicated earlier, this should be replace with a way to select the `feature-group` metrics and a way to select the "target" metrics.